Firestore/Source/Local/FSTLevelDB.mm

```diff
@@ -16,6 +16,8 @@
 
 #import "Firestore/Source/Local/FSTLevelDB.h"
 
+#include <dirent.h>
```

Comment:
These headers aren't portable.

Better to allow filesystem.h to provide `off_t` however it's going to get it.

Reply:
Removed. I think they were from a previous iteration of this anyways, and are no longer needed.